test_util/main.tf

```diff
@@ -74,7 +85,7 @@ module "dcos" {
 
   dcos_variant              = "${var.variant}"
   dcos_version              = "2.1.0-beta1"
-  dcos_license_key_contents = "${file("./license.txt")}"
+  dcos_license_key_contents = "${file("${var.license_file}")}"
```

Comment:
@fatz, should this be a file path or should we rather use the content and remove `file`?

Reply:
so from CI perspective plain text is easier to filter from output. Just thinking of Jenkins secret text vs secret file